frontend/packages/dev-console/src/components/topology/TopologySideBar.tsx

```diff
@@ -32,7 +34,14 @@ const TopologySideBar: React.FC<TopologySideBarProps> = ({ item, show, onClose }
     );
     const routes = metadataUIDCheck(item.resources.filter((o) => o.kind === 'Route'));
     const services = metadataUIDCheck(item.resources.filter((o) => o.kind === 'Service'));
-    const buildConfigs = metadataUIDCheck(item.resources.filter((o) => o.kind === 'BuildConfig'));
+    let buildConfigs = metadataUIDCheck(item.resources.filter((o) => o.kind === 'BuildConfig'));
+    buildConfigs = _.map(buildConfigs, (buildconfig: BuildConfigResourceProps) => {
+      return {
+        ...buildconfig,
+        builds: sortBuilds(buildconfig.builds),
```

Comment:
I think this should be done in topology-utils in the getBuildConfigs method. WDYT?

Reply:
Yes my initial thought was to add this logic in `getBuildConfigs` but then I thought shouldn't it depend upon the component using it to change the order according to its need. Anyways most of the use-cases will require the latest build first. I will add it in the `getBuidConfigs`.

Reply:
Done.